test/test-pages/missing-paragraphs/expected-metadata.json

```diff
@@ -0,0 +1 @@
+{"title":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy\n            eirmod tempor invidunt","byline":null,"excerpt":"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy\n            eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam\n            voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet\n            clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit\n            amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam\n            nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,\n            sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.\n            Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor\n            sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed\n            diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat,\n            sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.\n            Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor\n            sit amet."}
```

Comment:
Disclaimer: I don't know anything about anything, but why all the newlines (`\n`) in the title and excerpt?